### DISCUSSION TITLE
Question: Auto plugin updates

### DISCUSSION THREAD

**Discussion by quick007** [Minehut/Meta#526] [Q&A] [answered]
So every time someone requests a plugin on minehut, say it has thousands of lines of code, you read through every single line and make sure its safe? Then the next bug fix build for 3 plugins comes out the next day to fix crashes or smth and every single line changed is rereviwed? @pop4959 Is it like github where you can see the changed content, or do you manually just skim it?

**Comment by pop4959** [comment, marked as answer]
Answered in more detail on Discord, but the short answer is yes we do thoroughly review plugins on the platform.
